pkg/testcase/pod.go

```diff
@@ -56,4 +77,27 @@ func processPodStatus(
 			podAssertStatus(g, pod)
 		}
 	}
+
+	// at least one cilium pod should be running.
+	if ciliumPod {
```

Comment:
Wondering if we can move this code block inside `processCiliumStatus` to scope it within that func, that way we would not require the global vars on this file (which does not have any usage except for cilium based tests)

Reply:
and on thiws func, we need a way to only run this when it has cilium pods